src/tribler-core/tribler_core/components/restapi/rest/events_endpoint.py

```diff
@@ -74,7 +74,9 @@ def __init__(self):
         self.undelivered_error: Optional[dict] = None
 
         # We need to know that Tribler completed its startup sequence
+        # FIXME: remove this
```

Comment:
**Question:** I'm not sure sure what should be removed here and why.

Reply:
outdated...